cli/run_util.go

```diff
@@ -85,7 +85,6 @@ func (r WorkflowRunner) activateAndRunSteps(
 
 	// Global variables for restricting Step Bundle's environment variables for the given Step Bundle
 	currentStepBundleUUID := ""
-	// TODO: add the last step bundle's envs to environments
```

Comment:
This comment seemed obsolete, we even have a test case to check if the last step bundle's output env vars are available: https://github.com/bitrise-io/bitrise/blob/master/cli/run_test.go#L708